modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,68 @@
+
+== Control planes, discovery selectors, and namespace access
+
+Both control planes must have access to all namespaces in the mesh. During the migration, some proxies are controlled by the 3.0 control plane while other proxies remain controlled by the 2.6 control plane. To ensure that mesh communication works during the migration both control planes must be aware of the same set of services. Service discovery is provided by `istiod` component, which runs in the control plane namespace.
+
+In the {SMProduct} 3.0 installation, you can control how {istio} discovers services using discovery selectors. When using discovery selectors, ensure that the `discoverySelectors` expression defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. You might have to add additional labels to the {SMProduct} 2.6 application namespaces to ensure that they are captured in the {SMProduct} 3.0 installation. See "Scoping the service mesh with DiscoverySelectors" for more information.
+
+[NOTE]
+====
+In {SMProduct} 2.6 installations, the `maistra.io/member-of` label is automatically created. This label automatically gets removed during the migration process so it cannot be used.
+====
+
+== Control planes and network policies
+
+During the migration, for both control planes, ensure that no network policies block traffic between the control plane and the data plane namespaces, between the data plane namespaces and the control plane, or between the data plane namespaces themselves. By default, {SMProduct} 2.6 manages network policies that block traffic to the 3.0 control plane. 
+
+In the pre-migration checklist, you are instructed to disable network policies. However, you can manually re-create them. Manually created network policies must allow traffic for both control planes. When a data plane namespace is migrated to 3.0, the `maistra.io/member-of` label automatically gets removed. Do not use this label in network policies. See "Migration of Network Policies" for more information.
+
+[NOTE]
+====
+If network policies are configured incorrectly they can disrupt mesh traffic. When performing the migration, exercise caution when creating network policies to prevent traffic disruption. See "Network Policy migration" for more information.
+====
+
+== Control planes and sidecar injection
+
+During the migration, only one control plane must perform sidecar injection. This is achieved by correctly using injection labels. See "Installing the Sidecar" for more information.
+
+[NOTE]
+====
+During the migration, you must disable the 2.6 injector. Use the `maistra.io/ignore-namespace: "true"` label to prevent the 2.6 control plane from injecting a proxy in the namespace.
+====
+
+If both control planes try to perform sidecar injection the proxy will not start and the migration cannot be completed.
```

Comment:
```suggestion
If both control planes try to perform sidecar injection the proxy will not start and the migration cannot be completed. To ensure that only one control plane performs sidecar injection during the migration, use injection labels. For more information, see "Installing the Sidecar".

[NOTE]
====
During the migration, you must disable the 2.6 injector. Use the `maistra.io/ignore-namespace: "true"` label to prevent the 2.6 control plane from injecting a proxy in the namespace.
====
```
Should there be an xref somewhere after the module to the referenced doc?

Reply:
done